### PR TITLE
feat(context): add context providers to handle debug specific stuff

### PR DESCRIPTION
**Pull request by christianalfoni**
~~BREAKING CHANGE:
depends on Cerebral 0.34~~

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -48,9 +48,10 @@
     "validate-commit-msg": "^1.1.1"
   },
   "peerDependencies": {
-    "cerebral": "^0.33.8 || ^0.34.0-0"
+    "cerebral": "^cerebral@0.34.0-rc.2"
   },
   "dependencies": {
+    "cerebral-provider-mutationstracker": "^0.1.0",
```
Comment (Guria) [inline package.json]: remove it

Comment (christianalfoni) [inline package.json]: jup, will do

File: package.json [hunk 2]
```diff
@@ -35,7 +35,7 @@
     "analyzeCommits": "./sr-prerelease.js"
   },
   "devDependencies": {
-    "cerebral": "^0.33.31",
+    "cerebral": "^0.34.0-rc.2",
```
Comment (Guria) [inline package.json]: '^0.33.31 || ^0.34.0-rc.2'

File: package.json [hunk 3]
```diff
@@ -48,7 +48,7 @@
     "validate-commit-msg": "^1.1.1"
   },
   "peerDependencies": {
-    "cerebral": "^0.33.8 || ^0.34.0-0"
+    "cerebral": "^0.34.0-rc.2"
```
Comment (Guria) [inline package.json]: `"^0.33.31 || ^0.34.0-rc.2"`

**Comment by Guria** [comment]
LGTM

File: package.json [hunk 4]
```diff
@@ -48,7 +48,7 @@
     "validate-commit-msg": "^1.1.1"
   },
   "peerDependencies": {
-    "cerebral": "^0.33.8 || ^0.34.0-0"
+    "cerebral": "^0.33.31 || ^0.34.0-rc.2"
```
Comment (Guria) [inline package.json]:
``` js
"peerDependencies": {
  "cerebral": "^0.33.31 || ^0.34.0-rc.4"
},
```

**Pull request merged**